packages/core/src/index.html

```diff
@@ -175,11 +175,16 @@
         <meta name="dcterms.rightsHolder" content="Deriv" />
         <meta name="dcterms.rights" content="Deriv" />
         <meta name="google-site-verification" content="g5u3TyEdmYJ_HKZ6EmpW1nlKwD4XPVeLMhN9leeMfuw" />
-        <meta property="og:description" content="" />
+        <meta
+            property="og:description"
+            content="Deriv gives everyone an easy way to participate in the financial markets. Trade with as little as $1 USD on major currencies, stocks, indices, and commodities."
+        />
         <meta property="og:title" content="Deriv" />
         <meta property="og:type" content="website" />
-        <meta property="og:image" content="" />
-        <!-- TODO: Use og_image.gif once the new design is ready -->
+        <meta
+            property="og:image"
+            content="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTR3qsyDtQ_1mlX1AoeV963gFP0UZBTcJOcdK0G802Vfg&s"
+        />
```

Comment:
Preview link was only showing the text `Deriv`. Needed to update it so we can see a proper preview 

<img width="675" alt="Screenshot 2023-08-24 at 1 16 52 PM" src="https://github.com/ameerul-deriv/deriv-app/assets/103412909/a55a3ee5-df84-46cb-99b8-1d4a3a601ddd">

Reply:
couldn't add the image via relative path as its not supported for og:image, so the image was taken from deriv com